lib/ember_cli/runner.rb

```diff
@@ -32,9 +35,11 @@ def run!(command)
 
     private
 
-    def write(output, streams:)
-      streams.each do |stream|
-        stream.write(output)
+    def redirect(stream, pipes:)
+      Tee.open(*pipes, stdout: nil) do |pipe|
+        while line = stream.gets
```

Comment:
@jferris thanks for the suggestion.

I've tried `copy_stream` before, but not from a background thread -- is the Thread necessary?